Geometry/MTDNumberingBuilder/src/GeometricTimingDet.cc

```diff
@@ -223,5 +223,6 @@ GeometricTimingDet::Rotation GeometricTimingDet::rotationBounds() const {
 std::unique_ptr<Bounds> GeometricTimingDet::bounds() const {
   const std::vector<double>& par = params_;
   TrackerShapeToBounds shapeToBounds;
-  return std::unique_ptr<Bounds>(shapeToBounds.buildBounds(shape_, par));
+  return std::unique_ptr<Bounds>(
+      shapeToBounds.buildBounds(cms::dd::value_reverse(cms::LegacySolidShapeMap, shape_), par));
```

Comment:
it seems unnecessary here since the legacy solids would be read from the old DB payloads only.

Reply:
currently the 'shape_' here is still a `DDSolidShape` - so this is needed to be able to use `buildBounds` which  now expects a `cms::DDSolidShape` (or we change it to `int` - with the caveat that 7&8 don't exist in the new enum)